samples/TestService/project.json

```diff
@@ -10,6 +10,14 @@
         "Microsoft.Extensions.Configuration.CommandLine": "1.1.0"
     },
     "frameworks": {
+        "netcoreapp1.0": {
```

Comment:
We should probably keep the sample at the latest framework only. While project.json-based tooling automatically picks a framework for `dotnet run`, the new msbuild tooling will need an extra `-f netcoreapp1.*` argument at the moment, which is not a nice onboarding experience when you first try to run a sample..
To make sure the lib restores and compiles for lts+current .net core versions, i did this for the test project so tests are built and run for both frameworks.